app/views/posts/index.html.erb

```diff
@@ -1,9 +1,44 @@
-<!DOCTYPE html>
-<html>
-<head>
-  <title>All Posts by User</title>
-</head>
-<body>
-  <h1>Here is a list of posts for a given user</h1>
-</body>
-</html>
\ No newline at end of file
+<h1>Here is a list of posts for a given user</h1>
+
+<%= link_to user_path(@user), class: "user-row-link" do %>
+  <div class="user-photo">
+    <%= image_tag @user.photo %>
+  </div>
+  <div class="user-row">
+    <div class="user-info">
+      <h2><%= @user.name %></h2>
+      <div class="post-counter">Number of posts: <%= @user.posts.count %></div>
+    </div>
+  </div>
+<% end %>
+
+<% if @user.posts.any? %>
+  <h2>User's Posts</h2>
+  <% @user.posts.each do |post| %>
+    <%= link_to user_post_path(@user, post), class: "post-link" do %>
+      <div class="post-row">
+        <h3><%= post.title %></h3>
+        <p><%= truncate(post.text, length: 50) %></p>
+        <div class="post-stats">
+          <span>Comments: <%= post.comments.count %></span>
+          <span>Likes: <%= post.likes.count %></span>
+        </div>
+      </div>
+    <% end %>
+  <% end %>
+<% else %>
+  <p>No posts available for this user.</p>
+<% end %>
+
+<div class="comments-container">
+  <% @comments.each do |comment| %>
+    <div class="comment">
+      <span class="comment-author"><%= comment.user.name %>:</span>
+      <span class="comment-text"><%= comment.text %></span>
+    </div>
+  <% end %>
+</div>
```

Comment:
- [ ] Please kindly ensure that Users' recent_comments are renderered in the post_index page in accordance to the wireframe. ℹ️ 

<img width="284" alt="Untitled" src="https://github.com/arnoldnekemiah/blog/assets/103386183/c9a1a2c1-1c7b-44ec-b9bd-a63fb22bc0f1">

**Kindly take note that this request would have beed raised in the previous review but the reviewer was unable to access the post_inde page as highlighted in the inline comment. Thanks.**